.drone.yml

```diff
@@ -172,6 +172,20 @@ pipeline:
     when:
       event: [ push, tag, pull_request ]
 
+  test-mssql:
```

Comment:
before the test steps (look at group test) you could have a step that creates the DB like:

```
mssql-create-db:
   image: microsoft/mssql-tools
   commands:
    - sqlcmd -S mssql -U sa -P MwantsaSecurePassword1 -Q "CREATE DATABASE master"
   when:
      event: [ push, tag, pull_request ]
```

I haven't tested the above (or indentation) but something like it should work

Reply:
Just add 
```bash
sqlcmd -S mssql -U sa -P MwantsaSecurePassword1 -Q "CREATE DATABASE master"
```
to `commands` at `test-m$sql`.

Reply:
The better option might be to create the database in code like we do for pgsql and mysql:
https://github.com/go-gitea/gitea/blob/master/integrations/integration_test.go#L110

Reply:
I've implemented it and the test works now